azkaban-common/src/main/java/azkaban/executor/container/ContainerizedDispatchManager.java

```diff
@@ -505,6 +501,17 @@ public void run() {
     }
   }
 
+  @Override
+  protected void finalizeFlow(final ExecutableFlow flow, final String reason,
```

Comment:
Same here.
This one is "killing the corrresponding pod and mark the flow as finished in DB".
Also do you know why the original throwable can be null?